src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/response/LrdOrgInfoServiceResponse.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.response;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Objects;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class LrdOrgInfoServiceResponse {
+
+    private Long serviceId;
+
+    private String orgUnit;
+
+    private String businessArea;
+
+    private String subBusinessArea;
+
+    private String jurisdiction;
+
+    private String serviceDescription;
+
+    private String serviceCode;
+
+    private String serviceShortDescription;
+
+    private String ccdServiceName;
+
+    private String lastUpdate;
+
+    private List<String> ccdCaseTypes;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
```

Comment:
I think use lombok annaiotaion @hashcode and equals

Reply:
thanks @abhijitdiwan . Actually, these are not needed now. There was a bug in LRD api where it returns duplicate data. So I have written these codes to handle those duplicates. Now that has been fixed in LRD so this code is not necessary.